www/styles/admin-swat-local.css

```diff
@@ -948,4 +956,10 @@ div.swat-progress-bar-empty {
 		background-size: 22px 22px;
 	}
 
+	.swat-message-display-dismiss-link,
+	.swat-message-display-dismiss-link:link,
+	.swat-message-display-dismiss-link:visited {
+		background: url(../images/admin-message-display-dismiss@2x.png);
+		background-size: 18px 18px;
```

Comment:
The background-size values have to include the full height of the entire image sprite, not just the portion that is displayed. As it stands, this squishes both normal and hover versions of the "X" image into 18px of height.